modules/tensor_mechanics/doc/content/documentation/systems/BCs/Pressure/PressureAction.md

```diff
@@ -14,18 +14,25 @@ The Pressure Action is used to construct only the [Pressure boundary condition](
 !table id=pressure_BC_action_table caption=Correspondence Among Action Functionality and MooseObjects
 | Functionality     | Replaced Classes   | Associated Parameters   |
 |-------------------|--------------------|-------------------------|
-| A pressure traction force | [Pressure BC](/BCs/Pressure.md) | `displacements` : a string of the displacement variables to which the Pressure BC should be applied |
+| A pressure traction force given by a function| [Pressure BC](/BCs/Pressure.md) | `displacements` : a string of the displacement variables to which the Pressure BC should be applied |
+|  |  | `function` : a function that prescribes the pressure to be applied |
+| A pressure traction force given by an auxiliary variable | [Coupled Pressure BC](/BCs/CoupledPressureBC.md) | `displacements` : a string of the displacement variables to which the Pressure BC should be applied |
+|  |  | `variable` : an auxiliary variable prescribing the pressure to be applied |
 
 The Pressure Action only applies the pressure traction in the same component direction as the `displacements` variables are listed.
 That is, for the argument `displacements = 'disp_x disp_y disp_z'`, the Pressure Action will create three separate Pressure BCs:
 
-  - For the variable `disp_x`, the parameter setting `component = 0` is used
-  - For the variable `disp_y`, the parameter setting `component = 1` is used
-  - For the variable `disp_z`, the parameter setting `component = 2` is used
+- For the variable `disp_x`, the parameter setting `component = 0` is used
+- For the variable `disp_y`, the parameter setting `component = 1` is used
+- For the variable `disp_z`, the parameter setting `component = 2` is used
 
 Note that the placement of the variables in the `displacements` string determines the value of the component.
 
-As in the [Pressure](/BCs/Pressure.md) boundary condition, the  magnitude of the `Pressure` boundary condition can be specified as either a scalar (use the input parameter `factor`) or a `function` parameter.
+The [Pressure](/BCs/Pressure.md) boundary condition can be prescribed by
+
+1. the magnitude of the `Pressure` using a scalar value (use the input parameter `factor`),
+1. a function (use `function` parameter), or
+1. auxiliary variable (use `pressure` parameter and supply the variable name).
```

Comment:
Why always `1.` here?

Reply:
MOOSEDocs will render this as 1., 2. and 3 (check for yourself when the doco is generated). The advantage of this is that if you rearrange or add, you do not have to renumber things...

Reply:
Well that's pretty neat!